src/components/manifold-subscription-list/components/ListCard.tsx

```diff
@@ -34,7 +38,9 @@ const ListCard: FunctionalComponent<ListCardProps> = ({
       {$(plan.cost)}
       <span class="ManifoldSubscriptionCreate__Cost__Suffix">/mo</span>
     </span>
-    <button class="ManifoldSubscriptionCreate__List__Button">Modify Subscription</button>
+    <a href={ctaHref} onClick={onCtaClick} class="ManifoldSubscriptionCreate__List__Button">
```

Comment:
Here what I _think_ we should do is render a button if there's no `ctaHref`, and render an anchor if there is one.

Reply:
Of course there's the unconditional analytics tracking though isn't there...

Reply:
Why would we want to render a button instead of an anchor?

And re: the analytics do you mean analytics would be messed up if we did this? Couldn't we use `onClick` on both a button and anchor?

Reply:
> Why would we want to render a button instead of an anchor?

Just because buttons and anchors have different purposes, so it's sort of a semantic + accessibility issue. An anchor with an href is supposed to perform a navigation, whereas if clicking an element is supposed to produce some sort of side effect, a button is the right element for that job. It's a question of what we're communicating with our html.

> Couldn't we use onClick on both a button and anchor?

Yes, we can and that's probably what we'd do, it just involves some duplication which is unfortunate.